src/less/scrolling-table.less

```diff
@@ -126,3 +126,16 @@
     position: absolute;
 }
 
+
+.columnDrag {
+    box-shadow: inset 0 0 1em 1em rgba(0, 0, 0, 0.6);
```

Comment:
The 0.6 should be something a LOT more muted.  I took a look at using 0.1 or 0.15 and the effect is still seen without it being a black "mess".  That value itself should be a variable since we'll want to change it if you are in a dark "theme" to be a lighter highlight.